cloudify_gcp/compute/instance.py

```diff
@@ -249,26 +274,27 @@ def list(self):
 
         return self.discovery.instances().list(
             project=self.project,
-            zone=self.zone).execute()
+            zone=basename(self.zone)).execute()
 
     def to_dict(self):
         def add_key_value_to_metadata(key, value, body):
             body['metadata']['items'].append({'key': key, 'value': value})
 
-        def get_instance_ssh_keys():
-            agent_key = utils.get_agent_ssh_key_string()
-            other_keys = ctx.instance.runtime_properties.get(constants.SSHKEY)
-            return other_keys + '\n' + agent_key if other_keys else agent_key
+        network = {'network': 'global/networks/default'}
+        if self.network and self.network != 'default':
+            network['network'] = self.network
+        if self.subnetwork:
+            network['subnetwork'] = self.subnetwork
```

Comment:
Yeah, as per my last comment, I didn't really realise this version of the EphemeralIP functionality was here. I think just using a property on the node makes more sense than the `ExternalIP` node & relationship stuff though.